content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,355 @@
+       characteristics for the Uruguayan localization, which represent the minimum configuration
+       required for a company to operate in Uruguay according to the guidelines set by the |DGI|.
+       The module's installation automatically loads: chart of accounts, taxes, documents types, and
+       tax supported types.
+   * - :guilabel:`Uruguay Accounting EDI`
+     - `l10n_uy_edi`
+     - Includes all the technical and functional requirements to generate and validate
+       :doc:`Electronics Documents <../accounting/customer_invoices/electronic_invoicing>`, based on
+       the technical documentation published by the |DGI|. The authorized documents are :ref:`listed
+       above <uruguay/intro>`.
+
+.. note::
+   Odoo automatically installs the base module *Uruguay - Accounting* when a database is installed
+   with `Uruguay` selected as the country. However, to enable electronic invoicing, the *Uruguay
+   Accounting EDI* (`l10n_uy_edi`) module needs to be manually :ref:`installed <general/install>`.
+
+Configure your company
+----------------------
+
+To configure your company information, go to the :menuselection:`Settings --> Companies: Update
+Info` and configure the following:
+
+- :guilabel:`Company Name`
+- :guilabel:`Address`:
+
+  - :guilabel:`Street`
+  - :guilabel:`City`
+  - :guilabel:`State`
+  - :guilabel:`ZIP`
+  - :guilabel:`Country`
+
+- :guilabel:`Tax ID`: Enter the identification number for the selected taxpayer type.
+- :guilabel:`DGI Main Branch Code`: This is part of the XML when creating an electronic document. If
+  this field is not set, all electronic documents will be rejected.
+
+  To find the *DGI Main Branch Code*, follow these steps:
+
+  #. From your :ref:`Uruware account <l10n_uy/uruware-account>`, go to :menuselection:`Servicios en
+     línea DGI --> Registro único tributario --> Consulta de datos`.
+  #. Then, select :menuselection:`Consulta de Datos Registrales --> Consulta de Datos de Entidades`.
+  #. Next, open the generated PDF to get the *DGI Main Branch Code* from the :guilabel:`Domicilio
+     Fiscal Número de Local` section.
+
+After configuring the company in the database settings, navigating to :menuselection:`Contacts` and
+search for your company to verify the following:
+
+- Company type is set to :guilabel:`Company`.
+- Verify the :guilabel:`Identification Number` :guilabel:`Type` is :guilabel:`RUT / RUC`.
+
+.. _l10n_uy/uruware-account:
+
+Set up Uruware account
+----------------------
+
+To generate electronic documents in Odoo, a Uruware account is necessary.
+
+To set up a Uruware account, first be sure to have a valid Odoo subscription. Then follow the
+proceeding steps:
+
+Locate the Uruware credentials settings by navigating to the :menuselection:`Accounting -->
+Configurations --> Settings --> Localización Uruguaya`. Take a note of the admin email address that
+is set, this will be the same email used in Uruware's portal. Click on :guilabel:`Create Uruware
```

Comment:
Hello @samueljlieber and @ewa-odoo 

I suggest to add a tip box with the following "Enter the email of the administrator or project leader. Avoid using the same email as the one used for communication with the DGI" or add it in parenthesis after email used in Uruware's portal, this way users will know which email to use easier.

Cheers!

Fernando

@marianaluztorres